flink-cdc-cli/src/main/java/org/apache/flink/cdc/cli/CliFrontendOptions.java

```diff
@@ -53,12 +53,46 @@ public class CliFrontendOptions {
                     .desc("Use Flink MiniCluster to run the pipeline")
                     .build();
 
+    public static final Option SAVEPOINT_PATH_OPTION =
+            Option.builder("s")
+                    .longOpt("fromSavepoint")
```

Comment:
What about using `from-savepoint` to align with other options? Same issue for options below

Reply:
Addressed it.